apps/preview/next/tsconfig.json

```diff
@@ -32,7 +32,8 @@
       "@assets/*": [
         "./assets/fixture/*"
       ]
-    }
+    },
+    "incremental": true
```

Comment:
This line of config has been automatically updated for a few times during my work too. Should we just add it to the codebase?

Reply:
I will remove it, thx

Reply:
I think we should in one of the prs tho 😄 (it does not need to necessarily be this one)